tests/components/rflink/test_sensor.py

```diff
@@ -154,20 +185,40 @@
 
 async def test_race_condition(hass, monkeypatch):
     """Test race condition for unknown components."""
-    config = {"rflink": {"port": "/dev/ttyABC0"}, DOMAIN: {"platform": "rflink"}}
+    config = {
+        "rflink": {
+            "port": "/dev/ttyABC0"
+        },
+        DOMAIN: {
+            "platform": "rflink"
+        }
+    }
     tmp_entity = TMP_ENTITY.format("test3")
 
     # setup mocking rflink module
-    event_callback, _, _, _ = await mock_rflink(hass, config, DOMAIN, monkeypatch)
+    event_callback, _, _, _ = await mock_rflink(hass, config, DOMAIN,
+                                                monkeypatch)
 
     # test event for new unconfigured sensor
-    event_callback({"id": "test3", "sensor": "battery", "value": "ok", "unit": ""})
-    event_callback({"id": "test3", "sensor": "battery", "value": "ko", "unit": ""})
+    event_callback({
+        "id": "test3",
+        "sensor": "battery",
+        "value": "ok",
+        "unit": ""
+    })
+    event_callback({
+        "id": "test3",
+        "sensor": "battery",
+        "value": "ko",
+        "unit": ""
+    })
 
     # tmp_entity added to EVENT_KEY_SENSOR
-    assert tmp_entity in hass.data[DATA_ENTITY_LOOKUP][EVENT_KEY_SENSOR]["test3"]
+    assert tmp_entity in hass.data[DATA_ENTITY_LOOKUP][EVENT_KEY_SENSOR][
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4679175)